cfme/tests/cloud/test_tag_mapping.py

```diff
@@ -262,5 +243,21 @@ def test_ec2_tags_images():
             3. Go to summary of this image  and check whether there is
             test:testing in Labels field
             4. Delete that tag
+        testSteps:
+            1. Create an instance with tag test:testing
+            2. Refresh provider
+            3. Go to summary of this instance and check whether there is
+            test:testing in Labels field
+            4. Delete that instance
     """
-    pass
+    system = provider.mgmt
+    tag_key = f"test_{fauxfactory.gen_alpha()}"
+    tag_value = f"testing_{fauxfactory.gen_alpha()}"
+    if ec2taggable == EC2Image:
+        taggable = system.list_templates()[0]
+        request.addfinalizer(lambda: taggable.unset_tag(tag_key, tag_value))
+    else:
+        taggable = system.get_vm(testing_instance.name)
+    taggable.set_tag(tag_key, tag_value)
+    provider.refresh_provider_relationships(wait=600)
+    assert taggable.get_tag_value(tag_key) == tag_value
```

Comment:
Here `taggable` is a wrapanapi entity (either the image or the instance) by using `get_tag_value()` of the wrapanapi instance, this is not checking anything on the CFME side, but instead getting that info from EC2. We need to check if those tags show up in the CFME UI.